prime-router/src/main/kotlin/Element.kt

```diff
@@ -453,10 +462,22 @@ data class Element(
                     // this is a saving throw
                     val optionalDateTime = variableDateTimePattern
                     val df = DateTimeFormatter.ofPattern(optionalDateTime)
-                    val ta = df.parseBest(formattedValue, OffsetDateTime::from, LocalDateTime::from, Instant::from)
-                    LocalDateTime.from(ta).atZone(ZoneOffset.UTC).toOffsetDateTime()
+                    val ta = df.parseBest(
+                        formattedValue,
+                        OffsetDateTime::from,
+                        LocalDateTime::from,
+                        Instant::from,
+                        LocalDate::from
+                    )
+                    if (ta is LocalDateTime) {
+                        LocalDateTime.from(ta).atZone(ZoneId.of(USTimeZone.CENTRAL.zoneId)).toOffsetDateTime()
+                    } else {
+                        LocalDate.from(ta).atStartOfDay(ZoneId.of(USTimeZone.CENTRAL.zoneId)).toOffsetDateTime()
+                    }
                     return null
                 } catch (e: DateTimeParseException) {
+                } catch (e: DateTimeException) {
+                    // this could also happen
```

Comment:
Sometimes it's not just a DateTimeParseException. Sometimes it's also a DateTimeException.